src/main/java/codechicken/nei/ItemPanel.java

```diff
@@ -437,53 +464,23 @@ public ItemStack getStackMouseOver(int mousex, int mousey) {
         return slot == null ? null : slot.item;
     }
 
-    public ItemPanelSlot getSlotMouseOver(int mousex, int mousey) {
-        int index = firstIndex;
-        for (int i = 0; i < rows * columns && index < realItems.size(); i++)
-            if (validSlotMap[i]) {
-                if (getSlotRect(i).contains(mousex, mousey))
-                    return new ItemPanelSlot(index);
-                index++;
-            }
-
-        return null;
-    }
-
-    public void scroll(int i) {
```

Comment:
public method removal

Reply:
This method use to change active page. I don't think anyone use it

Reply:
I notice it's an old method (2014 CB) so I'd rather leave it in.  @deprecate it if you want, and if possible redirect, otherwise just empty it out.